app/AboutAppearanceViewController.m

```diff
@@ -158,4 +166,9 @@ - (IBAction)fontSizeChanged:(UIStepper *)sender {
     UserPreferences.shared.fontSize = @((int) sender.value);
 }
 
+- (void) setStatusBar:(id)sender {
+    [[UserPreferences shared] setShowStatusBar:((UISwitch *)sender).on];
```

Comment:
```suggestion
- (void)shouldHideStatusBarHiddenChanged:(UISwitch *)sender {
    UserPreferences.shared.hideStatusBar = sender.on;
```

Reply:
I instead changed it to `shouldHideStatusBarChanged` instead of having the extra "hidden in there" ;)

Reply:
It seems like you may have lost this change?

Reply:
ended up using `hideStatusBar` instead actually. Sorry if that's not exactly what was wanted though.

Reply:
To be clear, the method right now is `setStatusBar` and it takes an untyped parameter.

Reply:
OH, I was looking at the wrong file, jeez. Yeah you're right gimmie a sec.